nats-server/channelsz.yml

```diff
@@ -0,0 +1,40 @@
+type: json
+data:
+  url: /streaming/channelsz
+  server: ${NATS_SERVER_HOSTPORT}
+
+observation:
+  - name: channelsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: streaming.channels.total
+        source: total
+        type: long_gauge
+        label: total channels
+
+      - name: streaming.channels.max
+        source: limit
+        type: long_gauge
+        label: max channels
+
+  - name: channelsz_channel
+    metricNamespace: nats-streaming
+    path: $.channels[?(@.name=${channelName}})]
+
+    metric:
+      - name: streaming.channels.messages
+        source: msgs
+        type: long_gauge
+        label: messages
+
+      - name: streaming.channels.bytes
+        source: bytes
+        type: long_gauge
+        unit: bytes
+        label: bytes
+
+    tag:
+      - name: streaming.channels.name
```

Comment:
Tags must have `nats` prefix. In that case `nats.streaming.channels` would be a good name.

Reply:
DONE